taxcalc/tests/test_calculate.py

```diff
@@ -89,133 +69,130 @@ def test_sequence():
     run()
 
 
-# Create a basic Records object using Public Use File
-puf = Records(tax_dta)
-
-
 def test_make_Calculator():
-    # Create a Params object
-    params = Parameters(start_year=1991, inflation_rates=irates)
-    calc = Calculator(params, puf)
+    parm = Parameters()
+    assert parm.current_year == 2013
+    recs = Records(data=TAX_DTA, weights=WEIGHTS, start_year=2009)
+    calc = Calculator(params=parm, records=recs)
+    assert calc.current_year == 2013
 
 
 def test_make_Calculator_deepcopy():
     import copy
-    # Create a Params object
-    params = Parameters(start_year=1991, inflation_rates=irates)
-    calc = Calculator(params, puf)
-    calc2 = copy.deepcopy(calc)
+    parm = Parameters()
+    recs = Records(data=TAX_DTA, weights=WEIGHTS, start_year=2009)
+    calc1 = Calculator(params=parm, records=recs)
+    calc2 = copy.deepcopy(calc1)
 
 
 def test_make_Calculator_files_to_ctor(paramsfile):
     with open(paramsfile.name) as pfile:
         params = json.load(pfile)
     ppo = Parameters(parameter_dict=params, start_year=1991,
-                     num_years=len(irates), inflation_rates=irates)
-    calc = Calculator(params=ppo, records=tax_dta_path,
-                      start_year=1991, inflation_rates=irates)
+                     num_years=len(IRATES), inflation_rates=IRATES)
+    calc = Calculator(params=ppo, records=TAX_DTA_PATH,
+                      start_year=1991, inflation_rates=IRATES)
     assert calc
 
 
-def test_make_Calculator_mods():
-
-    # Create a Params object
-    params = Parameters(start_year=1991, inflation_rates=irates)
-
-    # Create a Public Use File object
-    puf = Records(tax_dta)
-
-    calc2 = calculator(params, puf, _II_em=np.array([4000]), _II_em_cpi=False)
+def test_make_Calculator_with_policy_reform1():
+    # create a Parameters object and apply a policy reform
+    params1 = Parameters()
+    reform1 = {2013: {'_II_em': np.array([4000]), '_II_em_cpi': False}}
+    params1.implement_reform(reform1)
+    # create a Calculator object using this policy-reform
+    puf = Records(data=TAX_DTA, weights=WEIGHTS, start_year=2009)
+    calc2 = Calculator(params=params1, records=puf)
+    # check that Parameters object embedded in Calculator object is correct
     assert all(calc2.params._II_em == np.array([4000]))
 
 
-def test_make_Calculator_json():
-
-    # Create a Params object
-    params = Parameters(start_year=1991, inflation_rates=irates)
-
-    # Create a Public Use File object
-    puf = Records(tax_dta)
-
-    user_mods = """{"1991": { "_STD_Aged": [[1500, 1250, 1200, 1500, 1500, 1200 ]],
-                     "_STD_Aged_cpi": false}}"""
-
-    calc2 = calculator(params, puf, mods=user_mods, _II_em_cpi=False,
-                       _II_em=np.array([4000]))
+def test_make_Calculator_with_policy_reform2():
```

Comment:
This shows the removal of the test of a capability to create a reform from a a JSON string. It turns out these reform dictionaries are serialized and deserialized to JSON in other parts of the web application, and so we haven't used this feature directly. @MattHJensen should sign off on the removal of this feature.